overview/roles.md

```diff
@@ -161,20 +161,16 @@ own standards)
 Quality Checkers (abbreviated *QC*) are often the last eyes on an
 episode before it is released.
 They are responsible for inspecting the
-overall quality of every aspect of a show including:
-timing,
-typesetting,
-encode,
-and editing.
+ensuring the overall quality of the release is up to par
+with the group's standards.
```

Comment:
While I don't think we necessarily need to mention each role specifically, I do think we should clarify on the "standards" part and make a reference to all of the previous tasks as a whole.

Reply:
Edited.

Reply:
Okay, `intricacies` is good for this.

However, could you reformat these sentences to use semantic linefeeds? They are explained in CONTRIBUTING.md.

Reply:
Like so?

Reply:
Good enough.